_dev/apps/ui/src/components/product-feed-page/disapproved-products-page/disapproved-products-page.vue

```diff
@@ -215,6 +216,14 @@ export default defineComponent({
       window.addEventListener('scroll', this.handleScroll);
     }
   },
+  beforeCreate() {
+    // We want to check the behavior of user with many disapproved products.
+    if (this.$store.getters['productFeed/GET_PRODUCT_FEED_VALIDATION_SUMMARY']
+      .disapprovedProducts > 250
```

Comment:
@sobartoli I think we haven't decided yet what are the conditions to trigger the recording

Reply:
Updated to match merchants with more than 100 products